sycl/test-e2e/Regression/same_unnamed_kernels.cpp

```diff
@@ -41,7 +41,9 @@ int main() {
         acc[0] *= 2;
       });
 
-  if (A[0] != 0 || B[0] != 2)
+  auto hostA = bufA.get_access<sycl::access::mode::read>();
+  auto hostB = bufB.get_access<sycl::access::mode::read>();
```

Comment:
```suggestion
  host_accessor hostA{bufA, sycl::read_only};
  host_accessor hostB{bufB, sycl::read_only};
```

Although I would also skip the `sycl::read_only` part and just use `host_accessor hostA{bufA};`

Reply:
thanks I've done this (Had some spurious test fails in other test though)